src/System.Private.CoreLib/shared/System/Threading/Tasks/Task.cs

```diff
@@ -6659,10 +6660,9 @@ private bool TrySetFromTask(Task task, bool lookForOce)
                 case TaskStatus.Faulted:
                     var edis = task.GetExceptionDispatchInfos();
                     ExceptionDispatchInfo oceEdi;
-                    OperationCanceledException? oce;
                     if (lookForOce && edis.Count > 0 &&
                         (oceEdi = edis[0]) != null &&
-                        (oce = oceEdi.SourceException as OperationCanceledException) != null)
+                        oceEdi.SourceException is OperationCanceledException oce)
```

Comment:
I don't see `oce` being used anywhere.

Reply:
It's used on the next line: `oce.CancellationToken`